charts/alfresco-search-enterprise/templates/reindexing-job.yaml

```diff
@@ -14,6 +15,7 @@ spec:
       {{- end }}
       labels:
         {{- include "alfresco-search-enterprise.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: {{ include "alfresco-search-enterprise.name" . }}
```

Comment:
this usage of selectorLabels instead of labels really looks like a bug

use https://github.com/gionn/helm-new-chart/blob/master/templates/deployment.yaml as a reference - it's the boiler template when asking helm to create a new chart